assets/src/admin/paired-browsing/app.css

```diff
@@ -0,0 +1,90 @@
+html,
+body {
+	margin: 0;
+	padding: 0;
+	height: 100%;
+	width: 100%;
+}
+
+body {
+	display: flex;
+	flex-direction: column;
+}
+
+body * {
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif;
+}
+
+#header {
+	height: 32px;
+	background-color: #0075c2;
+}
+
+#header * {
+	color: #fff;
+}
+
+#header ul {
+	display: flex;
+	margin: 0 10px;
+	padding: 0 40px;
+	list-style: none;
+	align-items: center;
+	height: 100%;
+}
+
+#header li {
+	margin: 0 5px;
+}
+
+#header li:last-child {
+	margin-left: auto;
+}
+
+#exit-link {
+	text-decoration: none;
+}
+
+#header img {
+	display: block;
+	width: 24px;
+}
+
+#amp::before {
+	content: "AMP";
```

Comment:
In case we do want to have the text visible (which I wouldn't recommend), we should at the very least add `pointer-events: none` to make sure they don't hinder interaction with the page.

Reply:
The initial thought when implementing this was that it may not be obvious to the user what each iframe is showing, so adding a distinct label for each should solve that. After reading your review and @swissspidy's however, I realize that this creates more problems than that of the one I intended to fix, so I will revert it for now.

With that said, do you think that the AMP admin bar item in each iframe acts as an adequate label? Maybe I'm overthinking this, so no worries either way :see_no_evil:  .